src/main/java/se/sundsvall/webmessagecollector/service/MessageService.java

```diff
@@ -56,6 +57,8 @@ public void deleteAttachment(final int id) {
 	}
 
 	public void deleteMessages(final List<Integer> ids) {
```

Comment:
Transactional?

Reply:
This will have transactional scope when called from resource class right? There should be no need to create a extra transaction.

Reply:
Ok, so there is a guarantee that this method will only be called from thr Resource-class?

(A Transactional-annotation without settings, will join the current transaction or create a new one if non exists. It will not creates another one, if not excplictly told.)

Reply:
There is no reference atm. elsewhere. There is no guarantee that there never will be. But for now the tranactions for messageService should be fine.